pom.xml

```diff
@@ -10,7 +10,7 @@
   </parent>
 
   <artifactId>cloudbees-folder</artifactId>
-  <version>5.19-SNAPSHOT</version>
+  <version>6.0.0-SNAPSHOT</version>
```

Comment:
https://github.com/jenkinsci/cloudbees-folder-plugin/pull/92/files#diff-73ee53b9a4352457ad14440d3f6692c0R282 (which I deem necessary) is the reason why I feel this is a bump to the major version.

I am using the bump to Major as a transition to a three digit scheme which will allow changes of this nature to be considered a minor bump